rarible/tests/exchange-v-1-utils.ts

```diff
@@ -0,0 +1,132 @@
+import { newMockEvent } from "matchstick-as"
+import { ethereum, Address, BigInt } from "@graphprotocol/graph-ts"
+import {
+  Buy,
+  Cancel,
+  OwnershipTransferred
+} from "../generated/ExchangeV1/ExchangeV1"
+
+export function createBuyEvent(
+  sellToken: Address,
+  sellTokenId: BigInt,
+  sellValue: BigInt,
+  owner: Address,
+  buyToken: Address,
+  buyTokenId: BigInt,
+  buyValue: BigInt,
+  buyer: Address,
+  amount: BigInt,
+  salt: BigInt
+): Buy {
+  let buyEvent = changetype<Buy>(newMockEvent())
```

Comment:
Why are we using mocking here?

Reply:
It should be done in test files.

Reply:
it is autogenerated as I did not even touch the tests folder